bgcval2/bgcvaltools/makeMask.py

```diff
@@ -439,12 +442,12 @@ def makeMask(name, newSlice, xt, xz, xy, xx, xd, debug=False):
             'OffShelf',
     ]:
         #bathync = ncdfView("data/ORCA1bathy.nc",Quiet=True)
-        bathync = dataset("data/ORCA1bathy.nc", 'r')
+        bathync = dataset(os.path.join(paths.bgcval2_repo, "bgcval2/data/ORCA1bathy.nc"), 'r')
```

Comment:
This still hardwires the ORCA1bathymetry into bgcval2. It would be better to be more flexible here, maybe set the bathymetry file elsewhere?